TESTPLAN.md

```diff
@@ -0,0 +1,47 @@
+Manual testing of the GCloud Plugin for Eclipse.
+
+# Install the plugin
+
+1. Launch Eclipse 4.5.
+
+2. If necessary, remove the old version of the plugin.
```

Comment:
Figure out what should happen if we install the new version over top of the old plugin.